docs/configuration/radio/security.mdx

```diff
@@ -34,7 +34,7 @@ The public key(s) authorized to send administrative messages to this node. Only
 
 Acceptable values: `true` or `false`
 
-Enabling Managed Mode restricts access to all radio configurations via client applications. Radio configurations will only be accessible through the Admin channel. Ensure the Admin channel is functioning properly before enabling this mode to avoid being locked out.
+Enabling Managed Mode restricts access to all radio configurations via client applications. This is not required to be set in order to remotely administer the node. Radio configurations will only be accessible through the Admin control messages or the legacy Admin channel. Ensure the Admin control or legacy Admin channel is functioning properly before enabling this mode to avoid being locked out. 
```

Comment:
This reads a little confusing to me, how about something like:

Enabling Managed Mode blocks client applications from accessing all radio configurations. However, this setting is not required for remote node administration. Once enabled, radio configurations can only be accessed through Admin control messages or the legacy Admin channel.

Before enabling Managed Mode, verify that the Admin control or legacy Admin channel is working properly to prevent being locked out.

Reply:
We'll go with that. Might take me a couple of hours...

Reply:
Sorry, re-reading this, would it make more sense to switch it around?

>Enabling Managed Mode blocks client applications from accessing all radio configurations. Once enabled, radio configurations can only be accessed through Admin control messages or the legacy Admin channel. However, this setting is not required for remote node administration.

Reply:
Yep, makes more sense. I've changed Admin to Remote Admin, as this is the terminology I'm using on the new Remote-Admin page.